requirements.txt

```diff
@@ -11,4 +11,5 @@ requests>=2.23.0
 SQLAlchemy>=1.3.16
 mbdata==25.0.4
 sqlalchemy-dst>=1.0.1
-importlib-metadata>=3.10.0
\ No newline at end of file
+importlib-metadata>=3.10.0
+itsdangerous==2.0.1  # last version compatible with Flask 1.*
```

Comment:
our flask dependency here is `Flask>=1.1.2`, which means that BU does make it possible to install flask 2 (and therefore we could use a newer version of itsdangerous). Should we add an upper limit to flask + werkzeug in the meantime before we upgrade it to v2??

Reply:
I checked and we already pin Flask in all downstreams. So I think fine to leave as is for now.

Reply:
We need to pin some other deps like MarkupSafe as well so I guess fine to upper bound Flask too then.

Reply:
Reverted the change because tests fail due to some reason with Flask 1.* but we know it works in prod :/. I guess investigate later or we are upgrading to Flask 2 soon with which the test passes so I guess fine anyway.